design_documents/parametrizing_tests_with_fixed_values.md

```diff
@@ -128,7 +128,14 @@ fn my_test_generated(a: felt252) {
 While it is technically possible to use complex types in attributes instead of defining fixtures, handling of them in
 code would be quite problematic based on my limited research.
 
-At least initially, we do not want to support syntax with complex types. For example, this would not be allowed:
+Annotations do not seem to be type-checked at all.
```

Comment:
Why this is the problem, i don't understand. Can't we typecheck them?

Reply:
We can I think, requires changing the test collector and digging into the compiler internals, but should be possible

Reply:
Not relevant anymore